examples/simple_box/assets/settings.ron

```diff
@@ -33,21 +33,21 @@ Settings(
             packet_loss: 0.05
         )),
         transport: [
-            WebTransport(
+            /* WebTransport(
```

Comment:
We should not comment out the web transport configs in this example

Reply:
ah i agree!
my mistake for leaving it commented out, i forgot to uncomment